packages/base-driver/lib/basedriver/commands/index.js

```diff
@@ -9,22 +9,27 @@ import {TimeoutMixin} from './timeout';
 import {ExecuteMixin} from './execute';
 
 /**
- * Applies all the mixins to the `BaseDriverBase` class.
- * Returns a `BaseDriver` class.
- * @param {BaseDriverBase} Base
+ * Applies all the mixins to the `BaseDriverBase` class; returns a `BaseDriver` class definition.
+ * Each mixin is applied in the order it is listed here, and each type is a union with the previous.
+ *
+ * @template {Constraints} C
+ * @param {BaseDriverBase<C>} Base
+ */
+export const createBaseDriverClass = _.flow(
```

Comment:
😻

Reply:
what's really scary is when I get to use lodash/fp